src/sentry/migrations/0726_apitoken_backfill_hashes.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 5.0.6 on 2024-05-29 21:28
+
+import hashlib
+import logging
+
+from django.db import migrations, router
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+logger = logging.getLogger(__name__)
+
+
+def backfill_hash_values(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    ApiToken = apps.get_model("sentry", "ApiToken")
+    ControlOutbox = apps.get_model("sentry", "ControlOutbox")
+    try:
+        from sentry.models.outbox import OutboxCategory, OutboxScope
```

Comment:
Generally we prefer to import as little as possible here, so I'd also recommend copy/pasting these into this migration

Reply:
Hmm, so when I attempt to import `OutboxCategory` via `apps.get_models(...)` it's unable to import it. 

```
E   LookupError: App 'sentry' doesn't have a 'OutboxCategory' model.
```

When I look at the output of `apps.get_models()` I don't see anything from `sentry.models.outbox`. Is there a proper way to do this?

```
...
 <class 'sentry.models.notificationmessage.NotificationMessage'>,
 <class 'sentry.models.notificationsettingoption.NotificationSettingOption'>,
 <class 'sentry.models.notificationsettingprovider.NotificationSettingProvider'>,
 <class 'sentry.models.organizationaccessrequest.OrganizationAccessRequest'>,
 <class 'sentry.models.organizationmemberteam.OrganizationMemberTeam'>,
 <class 'sentry.models.organizationmemberteamreplica.OrganizationMemberTeamReplica'>,
 <class 'sentry.models.organizationonboardingtask.OrganizationOnboardingTask'>,
 <class 'sentry.models.orgauthtoken.OrgAuthToken'>,
 <class 'sentry.models.platformexternalissue.PlatformExternalIssue'>,
 <class 'sentry.models.processingissue.ProcessingIssue'>,
 <class 'sentry.models.processingissue.EventProcessingIssue'>,
 <class 'sentry.models.projectbookmark.ProjectBookmark'>,
...
```

Reply:
Scratch that, these are not in alphabetical order like I had assumed. There are some available from `sentry.models.outbox`. 

```
<class 'sentry.models.outbox.RegionOutbox'>,
 <class 'sentry.models.outbox.ControlOutbox'>,
```

So I looked deeper and realized `OutboxCategory` and `OutboxScope` aren't Django models, but just regular Python Enums. 🤦 

I assume those are safe to import directly?

Reply:
For these I actually literally meant to just copy/paste the code inside the migration. It just prevents them breaking if people change code around later